### PR TITLE
Introduce relative positioning to some elements --> main

### PR DESCRIPTION
**Pull request by YoungMasterGandalf**
The name of the branch is misleading -- I am just adding two new attributes to Button which are then utilized for correct scaling in the platform

### CODE REVIEW

File: pgwidget/pgwidget_core.py [hunk 1]
```diff
@@ -1613,6 +1605,8 @@ def __init__(self,pos,size,text,function=lambda *args:None,function_args=None,bo
         self.visibility_layer=100
         self.hover_color=hover_color
         self.hover_label_color=hover_label_color
+        self.fix_horizontal_position = fix_horizontal_position
+        self.fix_vertical_position = fix_vertical_position
```
Comment (Omastto1) [inline pgwidget/pgwidget_core.py:1609]: are these variables used somewhere?

Comment (Omastto1) [reply pgwidget/pgwidget_core.py:1609]: button.fix_horizontal_position + fix_vertical_position is not used anywhere in pgwidget

i would rather see something like 
```
def relocate_element_with_label(self, screen_size_offset, in_x_direction=None, in_y_direction=None)
    if in_x_direction is None:
        in_x_direction = not self.fix_horizontal_position\
...
```

Comment (DovaX) [reply pgwidget/pgwidget_core.py:1609]: @YoungMasterGandalf - These two new attributes seem to me invalid and should be rather in layout object than in the button definition - requires deeper discussion

**Review by DovaX — CHANGES_REQUESTED** [review]

**Comment by DovaX** [comment]
I must close this PR - it is wrong to have it here - it is related to the graphical context - i.e. GLC + ordering in it, not to the button themselves

**Pull request closed without merging**